PowerEditor/src/Notepad_plus.cpp

```diff
@@ -1351,46 +1351,29 @@ void Notepad_plus::removeEmptyLine(bool isBlankContained)
 	_findReplaceDlg.processAll(ProcessReplaceAll, &env, true);
 }
 
-void Notepad_plus::getMatchedFileNames(const TCHAR *dir, const vector<generic_string> & patterns, vector<generic_string> & fileNames, bool isRecursive, bool isInHiddenDir)
+void Notepad_plus::getMatchedFileNames(const TCHAR *dir, const vector<generic_string> & patterns, vector<generic_string> & fileNames, bool isRecursive, bool isInHiddenDir, const Progress* progress)

```

Comment:
*Resident const nazi here:* Why isn't this `const Progress* const progress` (note that the second `const` doesn't need to be changed in the header)